packages/components/src/components/Combobox/Combobox.stories.tsx

```diff
@@ -0,0 +1,243 @@
+import type { Meta, StoryFn } from "@storybook/react";
+import React, { useEffect, useState } from "react";
+import toLower from "lodash/toLower";
+import includes from "lodash/includes";
+import map from "lodash/map";
+import { useComboboxStore } from "@ariakit/react";
+import { Controller, SubmitHandler, useForm } from "react-hook-form";
+import find from "lodash/find";
+import { Box } from "../../primitives/Box";
+import { Label } from "../Label";
+import { HelpText } from "../HelpText";
+import { Button } from "../Button";
+import { Combobox, ComboboxItemProps, ComboboxProps } from "./Combobox";
+import { ComboboxInput } from "./ComboboxInput/ComboboxInput";
+import { ComboboxPopover } from "./ComboboxPopover/ComboboxPopover";
+import { ComboboxItem } from "./ComboboxItem/ComboboxItem";
+
+export default {
+  component: Combobox,
+  title: "Components/Combobox",
+} as Meta<typeof Combobox>;
+
+const items: ComboboxItemProps[] = [
+  { value: "1", label: "Option One" },
+  { value: "2", label: "Option Two" },
+  { value: "Option Three", label: "Option Three" },
+  { value: "Option Four", label: "Option Four" },
+];
+
+const Template: StoryFn<ComboboxProps> = ({ children, ...props }) => (
+  <Combobox {...props}>{children}</Combobox>
+);
+
+export const Default = Template.bind({});
+
+Default.args = {
+  items,
+  placeholder: "Select an option",
+};
+Default.parameters = {
+  docs: {
+    description: {
+      story: "The combobox will filter the items based on the input value.",
+    },
+  },
+};
+
+export const WithoutFilterItem = Template.bind({});
+WithoutFilterItem.args = {
+  ...Default.args,
+  filterItem: false,
+};
+WithoutFilterItem.parameters = {
+  docs: {
+    description: {
+      story: "The combobox will not filter the items based on the input value.",
+    },
+  },
+};
+
+export const WithOnChange = Template.bind({});
+WithOnChange.args = {
+  ...Default.args,
+  onChangeInput: (event) => {
+    // eslint-disable-next-line no-console
+    console.log(event.target.value);
+  },
```

Comment:
this works ok but when I click `show code` it shows the onChangeInput function as a void function.. do you know why that is? (same for onSelectItem etc)

<img width="1025" alt="image" src="https://github.com/user-attachments/assets/6a02ebcd-d962-48cf-9673-a35ba69bb795" />

Reply:
I couldn't find a reason for why it's showing like that, but I discovered this https://storybook.js.org/docs/6/writing-docs/doc-blocks/doc-block-source and changed the stories properly to be able to show the callback. Changed here https://github.com/Localitos/pluto/pull/1788/commits/ee1b9df47f822dda26d23cf9f418d26bfb003584